### PR TITLE
Meetup 40

### PR DESCRIPTION
**Pull request by Drakevr**
Prepare the announcement:

- [x] Find a date and a place
- [x] Rename `meetup-40.md` to `YYYYMMDD-meetup-40.md` in the meetups directory
- [x] Fill in a nice title
- [x] Fill in a summary of the presentation
- [x] Fill in the name of the speaker as a link to their GitHub account
- [x] Fill in a short biography of the speaker
- [x] Add references to resources about the main topic of the presentation under "Notes" as a bullet list
- [x] Prepare video announcement
- [x] Create the meetup.com event
- [x] Create the facebook event
- [x] Add the links here
- [x] When all above are checked off, merge this
- [ ] Create a git.io URL with code thessrb-40 to the permanent link of `YYYYMMDD-meetup-40.md`
- [ ] Add event to [SKGTech Calendar](http://skgtech.io/submit-event/)
- [ ] Go crazy announcing the meetup everywhere

### CODE REVIEW

**Comment by kootoopas** [comment]
Should I message Eleftherakis to request a space in City College? As for other meetup dates, I know for sure that node is on Thursday the 21st.

**Comment by Drakevr** [comment]
Tue, January 19 looks like a good date!
Tue, January 26 looks good too!

**Comment by kootoopas** [comment]
I vote for 26 since it'll mean less pressure for those that need to prepare a presentation.

**Comment by petros** [comment]
26 might also be a bit better because it will give us more days to prepare as @kootoopas said, but it's also far enough from the huge amount of food we ate during the holiday season.

**Comment by petros** [comment]
> Should I message Eleftherakis to request a space in City College?

We can definitely ask. I usually ask here by at-mentioning George directly.

@eleftherakis can we have the auditorium for Tue, January 26?

**Comment by petros** [comment]
I updated the body of the PR to have the more fresh task list. I have it in TextExpander I am afraid. I will add a file with the snippet code in the repository, if anyone wants to start using that.

**Comment by eleftherakis** [comment]
@amiridis Done! 7-9 pm, auditorium, Tue 26/1

**Comment by petros** [comment]
Thanks @eleftherakis!!!

**Comment by kootoopas** [comment]
@petros Haha, I agree! We should allow the holiday eating enthusiasts a sufficient "running miles" margin. :smile:
@eleftherakis Thanks professor! :sunglasses:

File: meetups/20150126-meetup-40.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
+#### Thessaloniki Ruby Meetup 40
+
+Share   | http://git.io/thessrb-40
+------- | ------------------------
+When    | Day 26 January 2016 @ 19:00 - 21:00
```
Comment (petros) [inline meetups/20150126-meetup-40.md]: s/Day/Tuesday

Comment (Drakevr) [inline meetups/20150126-meetup-40.md]: woopsie

**Comment by Drakevr** [comment]
I suppose we are in a kinda CFP stage ATM?

**Comment by petros** [comment]
> I suppose we are in a kinda CFP stage ATM?

I suppose so. I will definitely do a rubyzino about Rails 5. Other than that our options are open. Any ideas welcome.

**Comment by petros** [comment]
@ispyropoulos could you send me the following information man?

* Title
* Short summary
* Speaker name / GitHub handle / Twitter handle
* Short speaker bio - including Testributor bio and whatnot

:smile:

**Comment by petros** [comment]
Hey @thanpolas could you add the event to the calendar! Thanks!

**Pull request merged**